tests/test_tasks.py

```diff
@@ -558,9 +558,9 @@ def flaky_function():
         @flow
         def test_flow():
             future = flaky_function.submit()
-            return future.wait()
+            return future.wait(), ...
```

Comment:
Was this updating the flow run with the task run state? You can use `quote` if so.